031_SumfOfDigits.js

```diff
@@ -0,0 +1,5 @@
+numberOfDigits = (a) => {
```

Comment:
You are asked to find the sum of the digits

Reply:
Will correct

Reply:
corrected